src/client/content_mapblock.cpp

```diff
@@ -447,46 +449,46 @@ void MapblockMeshGenerator::getLiquidNeighborhood()
 	u8 range = rangelim(nodedef->get(c_flowing).liquid_range, 1, 8);
 
 	for (int w = -1; w <= 1; w++)
-	for (int u = -1; u <= 1; u++) {
-		NeighborData &neighbor = liquid_neighbors[w + 1][u + 1];
-		v3s16 p2 = p + v3s16(u, 0, w);
-		MapNode n2 = data->m_vmanip.getNodeNoEx(blockpos_nodes + p2);
-		neighbor.content = n2.getContent();
-		neighbor.level = -0.5 * BS;
-		neighbor.is_same_liquid = false;
-		neighbor.top_is_same_liquid = false;
-
-		if (neighbor.content == CONTENT_IGNORE)
-			continue;
+		for (int u = -1; u <= 1; u++) {
```

Comment:
I think it's perfectly valid to align the inner `for` with the outer `for` - this is an exception to indent-inner-code-blocks rule, and the code would still look clean while not tending towards line length limit as much.

Reply:
Agreed, aligned 'for's are allowed in MT. This is especially useful in mapgen triple-loops to avoid excessively indented code.
This really needs to be configured if possible.